src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonResourceTest.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.AssertionsForClassTypes
+import org.junit.jupiter.api.Test
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.security.test.context.support.WithMockUser
+import uk.gov.justice.hmpps.prison.executablespecification.steps.AuthTokenHelper.AuthToken
+
+@WithMockUser
+class PrisonResourceTest : ResourceTest() {
+
+  @Test
+  fun testPrisonBookingSummaryReturnsResponseWithValidUser() {
+    val token = authTokenHelper.getToken(AuthToken.RELEASE_DATE_MANUAL_COMPARER)
+
+    val response = testRestTemplate.exchange(
```

Comment:
odd for new tests to use rest template instead of webclient - we're gradually trying to rewrite all to use webclient

Reply:
That's been done :)